Add doc improvements related to environment and IdP configurations for Dev Portal (#1793)
wso2/docs-choreo-dev: en/pe-docs/docs/api-management/developer-portal/configure-identity-providers-for-developer-portal.md

# Configure Identity Providers for Developer Portal

You can manage which Identity Providers (IdPs) are available for consumers to authenticate and generate API keys in the Developer Portal.

> !!! Note -
>     For organizations created after **24th of April 2025**, the Choreo Built-in Identity Provider is configured by default. Organizations created before this date allow all configured identity providers to be used in the Developer Portal. 

Reviewer: ```suggestion
!!! Note -
Organizations created after **April 24th, 2025**, will have the **Choreo Built-in Identity Provider** configured by default. Organizations created before this date will continue to allow all configured identity providers to be used in the Developer Portal.
```
